### PR TITLE
Feature/client email

### PR DESCRIPTION
**Pull request by domenic99**
#6

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -47,3 +47,4 @@ gem 'active_model_serializers'
 
 gem 'devise'
 gem 'jwt'
+gem 'mailcatcher'
```
Comment (EvgeniiK) [inline Gemfile]: please move this gem to development collection

Comment (domenic99) [inline Gemfile]: ok

File: app/mailers/client_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# class to send mails to client
+class ClientMailer < ApplicationMailer
+  default from: 'taxi@taxi.com'
+
+  def welcome_email(order_id)
+    @driver = Driver.find(Order.find_by(id: order_id).driver_id).name
+    mail(to: 'email@email.ua', subject: 'Taxi')
```
Comment (EvgeniiK) [inline app/mailers/client_mailer.rb]: Maybe you need to send this email to somebody?

Comment (domenic99) [inline app/mailers/client_mailer.rb]: But it is sent to email@email.ua. Also I tested with mailcatcher, all seems working

Comment (rakvium) [inline app/mailers/client_mailer.rb]: @domenic99 you should send it to an actual client, not to `email@email.ua` :wink:

Comment (EvgeniiK) [inline app/mailers/client_mailer.rb]: exactly!

Comment (domenic99) [inline app/mailers/client_mailer.rb]: oh yes. my fail :smile:

File: spec/mailers/client_mailer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe ClientMailerMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (rakvium) [inline spec/mailers/client_mailer_spec.rb]: @domenic99 add some examples here.

File: app/controllers/orders_controller.rb [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # class for orders management
 class OrdersController < ApplicationController
   before_action :check_client_params, only: [:create]
+  # before_action :driver_logged_in, only: [:update]
```
Comment (rakvium) [inline app/controllers/orders_controller.rb]: @domenic99, please delete commented code.

Comment (domenic99) [inline app/controllers/orders_controller.rb]: ok

File: app/mailers/client_mailer.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# class to send mails to client
+class ClientMailer < ApplicationMailer
+  default from: 'taxi@taxi.com'
+
+  def welcome_email(order_id, email)
+    @driver = Driver.find(Order.find(order_id).driver_id).name
```
Comment (rakvium) [inline app/mailers/client_mailer.rb]: @domenic99, I think, it's better to use car's name (number) here.

**Comment by domenic99** [comment]
I have squashed all my commits into one locally and git suggests (also google says the same) me to force push, should I ? :)

**Comment by EvgeniiK** [comment]
Yes, you need to force push your branch. In your branches, you can do force push.

**Pull request merged**